File: launchdarkly/ruby/src/GetAllReleaseProgressionsForReleasePipelineExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::ReleasePipelinesBetaApi.new.get_all_release_progressions_for_release_pipeline(
        "projectKey_string", # project_key
        "pipelineKey_string", # pipeline_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling ReleasePipelinesBetaApi#get_all_release_progressions_for_release_pipeline: #{e}"
end

File: launchdarkly/php/src/GetAllReleaseProgressionsForReleasePipelineExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\ReleasePipelinesBetaApi(config: $config))->getAllReleaseProgressionsForReleasePipeline(
        project_key: "projectKey_string",
        pipeline_key: "pipelineKey_string",
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling ReleasePipelinesBetaApi#getAllReleaseProgressionsForReleasePipeline: {$e->getMessage()}";
}
